# Indices of the obstacle pixels
int32[] i_pixels
